update-models.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
+
+MODELS_LIB_DIR="${BASE_DIR}/vendor/models-lib";
+TARGET_DIR="${BASE_DIR}/src";
+GENERATED_DIRS=("${TARGET_DIR}/Models" "${TARGET_DIR}/Enums");
+
+mkdir -p "$(dirname "${MODELS_LIB_DIR}")";
+if [[ ! -d "${MODELS_LIB_DIR}" ]]; then
+  git clone https://github.com/openactive/models-lib "${MODELS_LIB_DIR}";
+fi
+
+(
+  cd "${MODELS_LIB_DIR}";
+  command -v nvm && nvm use;
+
+  if [[ $(git rev-parse --abbrev-ref HEAD) == "master" ]]; then
+    git pull;
```

Comment:
I hit a slight issue here when trying to run this again because my package.lock file had been changed by the npm install below the first time round. Don't know if it's worth fixing.

Reply:
This was due to nvm not being available in the context of the shell script, and the install being done on the wrong version. I've added in a node version check which should resolve this.

Reply:
Yup, much happier now when I re-run the script.